### ISSUE TITLE
Make facecam work in PIP

### ISSUE THREAD

**Issue by yury** [blinksh/blink#1665]
### Discussed in https://github.com/blinksh/blink/discussions/1661

<div type='discussions-op-text'>

<sup>Originally posted by **cslaviero** December 16, 2022</sup>
Hi, I was wondering if facecam could work in PiP mode while Blink is in background. My usecase (not exclusive) would be that of a professor using Blink to teach programming and computer science stuff, and moving back and forth from drawing in some whiteboard app to Blink. iPadOS16 has enabled developers to keep the webcam enabled while in background (or so) : https://developer.apple.com/documentation/avfoundation/capture_setup/accessing_the_camera_while_multitasking . </div>

**Comment by yury** [comment]
Will be in next version. (with Center Stage support)

**Comment by cslaviero** [comment]
I've just tested it but with no sucess. I was on iPadOS 16.3.1 and tested with 16.4 Beta 1.  I've tried the following:

- Open Blink app 
- launch ``` facecam ```  
- Using ```cmd+tab``` or drag to switch apps

expected behavior: PiP window would appear as soon as Blink went to background
current behavior: PiP window does not appear, and the camera feed seems to be shut down (it does not show being used on Control Center. 

PS: I'm on iPad Pro 2020.
